stable/harbor/app-readme.md

```diff
@@ -0,0 +1,10 @@
+# SUSE Private Registry powered by Harbor - Cloud Native Repository for Kubernetes
+
+[Harbor](https://github.com/goharbor/harbor) is an open source container image registry that secures images with role-based access control, scans images for vulnerabilities, and signs images as trusted. As a CNCF Incubating project, Harbor delivers compliance, performance, and interoperability to help you consistently and securely manage images across cloud native compute platforms like Kubernetes and Docker.
```

Comment:
Eh, I do not really know the process, but I believe it's past the incubation already

Reply:
You're right. Fixed it.